noisyatom/config/settings.py

```diff
@@ -41,16 +41,16 @@
     # DATABASES = {
     #     'default': {
     #         'ENGINE': 'django.db.backends.postgresql',
-    #         'NAME': 'na_db',
-    #         'USER': 'na_db_user',
-    #         'PASSWORD': 'na_user',
+    #         'NAME': 'ffw',
```

Comment:
I would suggest not putting your personal passwords of your DB into an open project. This will eventually be picked up by the new git_commit software test tools when we switch them on. But if this is just a test DB on your local machine then sure, i have no issues.